src/Raven.Client/Documents/Session/AsyncDocumentQuery.cs

```diff
@@ -270,13 +270,6 @@ public IAsyncDocumentQuery<T> WhereExists<TValue>(Expression<Func<T, TValue>> pr
             return this;
         }
 
-        /// <inheritdoc />
-        IAsyncDocumentQuery<T> IFilterDocumentQueryBase<T, IAsyncDocumentQuery<T>>.CmpXchg(string key, T value)
```

Comment:
I think that we should not remove that support from DocumentQuery

Reply:
Agreed